odoo/tools/config.py

```diff
@@ -764,11 +766,19 @@ def _check_path(cls, option, opt, value):
 
     @classmethod
     def _check_without_demo(cls, option, opt, value):
+        if value == 'False':
+            return False
+        else:
+            cls._warn(
+                "The without-demo option should be used as a flag.",
+                DeprecationWarning
+            )
```

Comment:
I don't see the point of those few lines, aren't
```py
        try:
            return not cls._check_bool(option, opt, value)
        except optparse.OptionValueError:
            cls._log(logging.WARNING, "option %s: since 19.0, invalid boolean value: %r, assume %s", opt, value, value != 'None')
```
enough?